version/K8S.VERSION

```diff
@@ -1 +1 @@
-v0.17.0-1.9.9
\ No newline at end of file
+v1.0.0-1.9.9
```

Comment:
Maybe we can remove this file, I remember this is for yarn version, and doesn't need for now.

Reply:
Remove it in the next release? I'm not sure if the removal will cause any problem.

Reply:
Sure, fine for me